dataset/pascal_voc.py

```diff
@@ -156,8 +186,9 @@ def _load_image_labels(self):
                 #     continue
                 cls_name = obj.find('name').text
                 if cls_name not in self.classes:
-                    continue
-                cls_id = self.classes.index(cls_name)
+                    cls_id = len(self.classes)
```

Comment:
Do you intended to regard everything not in provided names as one class?

Reply:
Currently this is the best I could think of ..
at first I thought maybe to list them as ignores ('-1') but it was not a good hack